releasenotes/notes/low-rank-linalg-86da86096d6122d8.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Added linear algebra utilities for performing the double-factorization of a two-body tensor:
+      - :func:`qiskit_nature.utils.double_factorized`
```

Comment:
```suggestion

      - :func:`qiskit_nature.utils.double_factorized`
```

Lets try adding a space (blank line) before the inner list. It looks better but I think its interpreting the whole thing as a definition list now where the first sentence is the term, hence the bold in the resultant html

![image](https://user-images.githubusercontent.com/40241007/235730354-3ab06813-9f57-4eb7-836a-2b937c3c863e.png)